proto/beacon/rpc/v1/services.proto

```diff
@@ -67,12 +67,13 @@ message ValidatorAssignmentRequest {
 
 message ValidatorAssignmentResponse {
     repeated Assignment assignments = 1;
-    uint64 slot = 2;
+    uint64 current_beacon_slot = 2;
```

Comment:
Why change this?

The validator reasonably knows what slot they are a currently in, and this value serves as the slot they are expected to be assigned.

Please revert this change, unless there is an important reason for this change.